.pre-commit-config.yaml

```diff
@@ -16,30 +18,36 @@ repos:
     -   id: reorder-python-imports
         files: ^src/|tests/
 
--   repo: local
+-   repo: https://github.com/pre-commit/mirrors-mypy.git
+    rev: v0.910
     hooks:
     -   id: mypy
-        name: mypy
-        entry: mypy src
-        language: system
-        pass_filenames: false
+        files: ^src/|tests/
+        # 😔 These `additional_dependencies` mirror those in the `dev` extra of `setup.cfg` and are needed
+        # to run `mypy` under `tox`. This violates "DRY" but we don't have a better way of doing this—yet.
+        additional_dependencies: [
+            lxml-stubs,
+            pandas-stubs,
+            types-flask,
+            types-jsonschema,
+            types-pkg_resources,
+            types-python-dateutil,
+            types-requests,
+            types-six,
+            types-waitress
+        ]
 
--   repo: local
+-   repo: https://github.com/python/black
+    rev: 22.3.0
     hooks:
     -   id: black
-        name: black
-        entry: black
         files: ^src/|tests/
-        language: system
-        types: [python]
 
--   repo: local
```

Comment:
Ditto; this should remain as `repo: local`. I think there was some overzealous copy/paste from `doi-service` when all this PR needed was the `exclude: REQUIREMENTS.md` for `end-of-file-fixer` and `trailing-whitespace`.